wme-sitebuilder/Wizards/GoLive.php

```diff
@@ -78,6 +87,48 @@ public function props() {
 		];
 	}
 
+	/**
+	 * Action: init
+	 *
+	 * Add rewrite rule to receive domain purchase webhook.
+	 */
+	public function action__init() {
+		add_rewrite_rule( '^' . self::REWRITE_TAG . '$', 'index.php?' . self::REWRITE_TAG . '=true', 'top' );
```

Comment:
If we're adding a rewrite rule, we'll likely need to flush permalinks. I don't believe we have a mechanism that tracks Sitebuilder version yet, but we may want to consider implementing one and scheduling or performing the flush when this is updated.

For now, we can rely on the StoreBuilder implementation itself to run the flush as it does have a method of handling data migration already.